node/Cargo.toml

```diff
@@ -3,158 +3,149 @@ name = "zeitgeist"
 path = "./src/main.rs"
```

Comment:
You seem to have autoformatted this file into oblivion. These changes are not consistent with the standard `.toml` style.